Automattic/portkey-android
# Add StoryLoadStart and StoryLoadEnd events

This PR adds two events that are triggered from method `onLoadFromIntent()`:
- `StoryLoadStart` is triggered right away upon method start, so any interested parties listening can track when the ComposeLoopFrameActivity is about to load a Story coming from any of the entry points set through [Intent extras](https://github.com/Automattic/stories-android/wiki/2.-API#entry-points)
- `StoryLoadEnd` is triggered in all cases within `onLoadFromIntent()` in which a Story has been loaded

**Note**: given we were adding new events which were similar to `StorySaveEvents` but of a different nature, I first created a new package to keep both classes separately for cleanliness (see c4382d8), but turns out given we're serializing the slides along with their save results on the WPAndroid app this meant previously saved stories would not be able to get deserialized after this change. So, given this, I decided to add the class right along the same package where the StorySaveEvents where.

To test:
- See accompanying WPAndroid PR https://github.com/wordpress-mobile/WordPress-Android/pull/15310, or
- alternatively: observe the desired event and make sure it gets delivered, by using a breakpoint.
